src/aims_ui/models/endpoint.py

```diff
@@ -0,0 +1,26 @@
+from flask import url_for
+
+
+class Endpoint():
```

Comment:
If this class is mostly/only storing data, a Python dataclass would produce cleaner code.

Reply:
Convertedto dataclass